recipes/redradist-icc/all/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+cmake_minimum_required(VERSION ${CMAKE_VERSION})
```

Comment:
Our defacto is 3.1, you do not need a variable

Reply:
How to figure out the lower bound ?
`STX` package use `3.5` https://github.com/conan-io/conan-center-index/blob/master/recipes/stx/all/CMakeLists.txt ...